### PR TITLE
Fix broken email sharing

### PR DESCRIPTION
**Pull request by VassilIordanov**
Add missing context argument to loadApprovalStatus in emailReport.ftl

fixes NCI-Agency/anet#865

### CODE REVIEW

File: src/main/resources/emails/emailReport.ftl [hunk 1]
```diff
@@ -469,8 +469,8 @@ ${sender.name} sent you a report from ANET:
     </div>
   </#if>
 
-  <#if report.loadApprovalStatus()??>
-    <#assign approvalStatus = report.loadApprovalStatus()>
+  <#if report.loadApprovalStatus(context)??>
```
Comment (maradragan) [inline src/main/resources/emails/emailReport.ftl]: I think you should use report.loadApprovalStatus(context).get(), as loadApprovalStatus returns a CompletableFuture.

File: src/main/resources/emails/emailReport.ftl [hunk 2]
```diff
@@ -469,8 +469,8 @@ ${sender.name} sent you a report from ANET:
...
+    <#assign approvalStatus = report.loadApprovalStatus(context)>
```
Comment (maradragan) [inline src/main/resources/emails/emailReport.ftl]: this is doing the same again, shouldn't we define a getApprovalStatus (like it has been done with loadLocation and getLocation for instance?) and use it in here?

Comment (VassilIordanov) [reply src/main/resources/emails/emailReport.ftl]: agreed, will push in 10

Comment (VassilIordanov) [reply src/main/resources/emails/emailReport.ftl]: hmm, I think we should get rid of the getXXX / loadXXX combo eventually, so I made a single call on the ftl side instead.

**Pull request merged**